scylla-cql/Cargo.toml

```diff
@@ -13,7 +13,7 @@ license = "MIT OR Apache-2.0"
 scylla-macros = { version = "0.2.0", path = "../scylla-macros"}
 byteorder = "1.3.4"
 bytes = "1.0.1"
-num_enum = "0.5"
+num_enum = "0.6"
```

Comment:
Note: this is a breaking change. The `TryFromPrimitive` from version 0.5 and 0.6 are not the same trait because those versions are not semver-compatible.